tests/includes/test-class-accredible-learndash-admin-action-handler.php

```diff
@@ -34,23 +34,38 @@ public function test_add_auto_issuance() {
 		// Login as an admin.
 		$admin_user = self::factory()->user->create( array( 'role' => 'administrator' ) );
 		wp_set_current_user( $admin_user );
-		$nonce        = wp_create_nonce( 'add_auto_issuance' );
-		$redirect_url = admin_url( 'admin.php?page=accredible_learndash_issuance_list&page_num=3' );
-
-		$this->expectOutputString( "<p>Processing...</p><script>window.location.href='$redirect_url'</script>" );
-		Accredible_Learndash_Admin_Action_Handler::add_auto_issuance(
+		$nonce         = wp_create_nonce( 'add_auto_issuance' );
+		$redirect_url  = admin_url( 'admin.php?page=accredible_learndash_issuance_list&page_num=3' );
+		$output_string = wp_json_encode(
 			array(
-				'nonce'                       => $nonce,
-				'redirect_url'                => admin_url( 'post-new.php' ),
-				'page_num'                    => 3,
-				'accredible_learndash_object' => $new_data,
+				'success' => true,
+				'data'    => array(
+					'message'     => 'Saved auto issuance successfully.',
+					'redirectUrl' => esc_url( $redirect_url ),
+				),
 			)
 		);
+
+		try {
+			Accredible_Learndash_Admin_Action_Handler::add_auto_issuance(
```

Comment:
I think we should also test the returning value if it doesn't raise an exception.

Reply:
Added missing assertion.